scripts/generate-sw.js

```diff
@@ -0,0 +1,8 @@
+var path = require('path');
+var swPrecache = require('sw-precache');
+var __dirname = 'themes/aframe/source';
+
+swPrecache.write(path.join('scripts','sw.js'), {
```

Comment:
Yeah, otherwise it won't be part of the site. You'd also have to add a script tag for it?